modules/ROOT/pages/reference/procedures.adoc

```diff
@@ -28,7 +28,7 @@ SHOW PROCEDURES
 Some procedures can only be run by users with `Admin` privileges.
 These are labeled with label:admin-only[].
 
-For more information, see link:{neo4j-docs-base-uri}/cypher-manual/{page-version}/administration/access-control/manage-privileges/[Cypher Manual -> Manage Privileges].
+For more information, see xref:authentication-authorization/manage-privileges.adoc[Cypher Manual -> Manage Privileges].
```

Comment:
if the link no longer points to Cypher Manual, should the link text also be updated?
```suggestion
For more information, see xref:authentication-authorization/manage-privileges.adoc[Manage Privileges].
```